python/proton/_handlers.py

```diff
@@ -322,74 +322,64 @@ def __init__(self, peer_close_is_error: bool = False, delegate: Optional[Handler
         self.peer_close_is_error = peer_close_is_error
 
     @classmethod
-    def is_local_open(cls, endpoint):
+    def is_local_open(cls, endpoint: Endpoint) -> bool:
         """
         Test if local ``endpoint`` is open (ie has state
         :const:`proton.Endpoint.LOCAL_ACTIVE`).
 
         :param endpoint: The local endpoint to be tested.
-        :type endpoint: Any child of :class:`proton.Endpoint`
         :return: ``True`` if local endpoint is in state
             :const:`proton.Endpoint.LOCAL_ACTIVE`, ``False`` otherwise.
-        :rtype: ``bool``
         """
-        return endpoint.state & Endpoint.LOCAL_ACTIVE
+        return bool(endpoint.state & Endpoint.LOCAL_ACTIVE)
```

Comment:
The return type used to be `int`; I think in this instance it makes sense to adjust the code to match the docs, rather than the other way around.